src/Raven.Client/Documents/Linq/LinqPathProvider.cs

```diff
@@ -60,6 +60,15 @@ public Result GetPath(Expression expression)
                         throw new ArgumentException("Not supported computation: " + callExpression +
                                             ". You cannot use computation in RavenDB queries (only simple member expressions are allowed).");
 
+                    if (isFilterActive)
+                    {
+                        return new Result
+                        {
+                            MemberType = callExpression.Method.ReturnType,
+                            IsNestedPath = false,
+                            Path = "Count"
```

Comment:
`nameof` ?

Reply:
Only Count is possible in this codepath.